src/align_assign.cpp

```diff
@@ -31,8 +31,10 @@ chunk_t *align_assign(chunk_t *first, size_t span, size_t thresh, size_t *p_nl_c
    }
    size_t my_level = first->level;
 
+   char   *copy = (char *)malloc(1000);
```

Comment:
This looks like a memory leak? How does this get `free`d? Maybe you want to use a stack-allocated buffer instead? Or, alternatively, just have the chunk manage the memory, allocated on use. I imagine chunks with more than 1KB of text are rare. (This would also simplify use, as you wouldn't need to worry about a buffer at each point of use.)